apps/frontend/src/modules/surveys/states/studio.ts

```diff
@@ -2,6 +2,7 @@ import { create } from "zustand";
 
 export type StudioState = {
     studioModeIsActive: boolean;
+    editTheme: boolean;
```

Comment:
@kaf-lamed-beyt Why do we need this state?

Reply:
@jdamilola, It is referenced here. 

https://github.com/IntegraflowHQ/integraflow/blob/df28ca66d3342d46170895f4a03b4b0203327e4d/apps/frontend/src/modules/surveys/components/studio/create/editor-panel/DesignEditor/index.tsx#L182-L201

I'm using a global state with Zustand, so we don't end up having race conditions from drilling props.

Reply:
The idea behind this is so, the UI of the color picker &mdash; the buttons, specifically &mdash; matches the action a user performs. 

If they try to click on the Pen icon, to edit a theme, both buttons &mdash; Update theme and Delete Theme &mdash; should be rendered. If not, one button should be rendered when they click on the "New Theme" button, and the text should be "Create Theme" since they'll be creating a new theme. You can't delete a theme that hasn't been created.

Reply:
I'm still working on the feedback you gave me yesterday.  By evening or Night, I should've updated this PR with the fix. If I don't find a headway... I'll let you know.